puma/README.md

```diff
@@ -0,0 +1,54 @@
+# Agent Check: puma
+
+## Overview
+
+This check monitors [puma][1] through the Datadog Agent.
```

Comment:
Can we include a brief description of Puma and what this integration can monitor?